src/snkit/network.py

```diff
@@ -317,17 +330,20 @@ def _split_edges_at_nodes(
 
 
 def split_edges_at_nodes(network, tolerance=1e-9):
-    """Split network edges where they intersect node geometries"""
+    """
+    Split network edges where they intersect node geometries. Will operate in
+    parallel if SNKIT_PROCESSES is in the environment and a positive integer.
+    """
     split_edges = []
 
     n = len(network.edges)
-    if PARALLEL and (n > 10_000):
-        chunk_size = int(n / os.cpu_count())
+    if PARALLEL_PROCESS_COUNT and (n > 1_000):
+        chunk_size = max([1, int(n / PARALLEL_PROCESS_COUNT)])
```

Comment:
I wonder if `1_000` (`threshold_for_parallelism`?) and/or this method for coming up with `chunk_size` are somewhat data-dependent? Not obviously a concern for most library users, but could be handy as optional arguments to `split_edges_at_nodes` if someone did ever need them.

Reply:
I gave the function an optional `chunk_size` arg. If it's `None` (default value), use the existing method to determine `chunk_size`. I've removed the threshold, seemed to be getting too clunky.